doc/helm-values.adoc

```diff
@@ -0,0 +1,381 @@
+= Helm Reference Table
+
+== Global Settings
+
+[cols="10,10,20a,50a"]
+|===
+|setting |default value |valid values |description
+
+|`drbdRepoCred`
+|`drbdiocred`
+|secret name
+|Names a secret containing registry credentials to pull LINSTOR container images,
+
+|`global.imagePullPolicy`
+|`IfNotPresent`
+|
+* `""`
+* `Always`
+* `IfNotPresent`
+* `Never`
+|Global pull policy to apply to all images. Can be set to `""` to use kubernetes default behaviour. See https://kubernetes.io/docs/concepts/containers/images/#updating-images[pull policy].
+
+|`linstorHttpsClientSecret`
+|`""`
+|secret name
+|References the secret to use when configuring LINSTOR clients to use HTTPS. Check out link:./security.md#configuring-secure-communications-for-the-linstor-api[the security guide]
+
+|`linstorHttpsControllerSecret`
+|`""`
+|secret name
+|References the secret to use when configuring LINSTOR contoller to use HTTPS. Check out link:./security.md#configuring-secure-communications-for-the-linstor-api[the security guide]
+
+|`priorityClassName` |`""` |https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass[priority class] name |Name of the priority class with which the LINSTOR components should be scheduled.
+|===
+
+== Operator (`operator` section)
+
+[cols="10,10,20a,50a"]
+|===
+|setting |default value |valid values |description
+
+|`image`
```

Comment:
At https://github.com/bitnami/charts/tree/master/bitnami/etcd they have the whole key which I found useful because then you can just copy it into a `--set` flag. That is less useful if you are writing a yaml values file, but I find it overall more explicit and easier to understand at a glance. What do you think?

Reply:
It started out with the full key, but github's rendering is even worse in that case